ast/compile_test.go

```diff
@@ -2068,7 +2068,7 @@ func TestCompilerCheckRuleConflictsDotsInRuleHeads(t *testing.T) {
 				`package pkg.p
 				default q.w.r = 4
 				q.w.r = 2`),
-			err: "rego_type_error: multiple default rules data.pkg.p.q.w.r found",
+			err: "rego_type_error: multiple default rules data.pkg.p.q.w.r found at mod0.rego:2, mod1.rego:2",
```

Comment:
I'm a little surprised that the ordering here is stable across test re-runs without sorting the `defaultRules` slice. But if that's the case, fine; we can fix it if it ever becomes a problem later.